### DISCUSSION TITLE
Version 2.1.3 UI Looks weird

### DISCUSSION THREAD

**Discussion by hpatel-higi** [apache/airflow#18252] [Q&A] [answered]
I just upgraded to version 2.1.3 with python 3.9-slim.  Noticed the UI looks a bit weird, things are getting cut off.  Checking the console of the browser and boostrap.min.css could not be loaded.

![image](https://user-images.githubusercontent.com/67915750/133321808-9b139b5f-97bd-4e8e-8632-75483c9e8590.png)

**Comment by potiuk** [comment, marked as answer]
Known issue. Will be solved in 2.1.4

**Reply by potiuk** [reply]
(which is going to be released today/tomorrow)
